mobile-widgets/qml/HintsTextEdit.qml

```diff
@@ -14,7 +14,11 @@ TextField {
 	onTextChanged: {
 		textUpdateTimer.restart();
 	}
-	onFocusChanged: frame.shouldShow = focus
+	onFocusChanged: {
+		focus != focus
```

Comment:
my point was that this line should have no effect. This line is just a statement that will evaluate to false.
It is NOT an assignment that inverts focus.